src/BlazorDB/wwwroot/blazorDB.js

```diff
@@ -32,6 +106,9 @@ window.blazorDB = {
 
             stores[schema.name] = def;
         }
+        if (dbStore.dynamic) {
```

Comment:
I don't think belongs in `createDb`.  Shouldn't it be handled in C# or in the `addSchema` function?